torch/_dynamo/variables/misc.py

```diff
@@ -712,6 +712,9 @@ def visit(node):
         ctx = AutogradFunctionContextVariable.create(tx, args, kwargs)
         args = [ctx, *args]
         if isinstance(fn, types.FunctionType):
+            sig = inspect.signature(fn)
+            if len(args) == len(sig._parameters) - 1:
```

Comment:
should be `len(args) == len(sig._parameters) + 1:`
or `len(args) -1 == len(sig._parameters):` right?

Reply:
lol yeah oops

Reply:
Fixed